www/app/blog/2025-05-09-uuidv7.mdx

```diff
@@ -0,0 +1,176 @@
+export const metadata = {
+  title: "UUIDv7 Comes to PostgreSQL 18",
+  authors: ["gwenshap"],
+  image: "2025-05-09-uuidv7/cover.png",
+  sizzle:
+    "PostgreSQL 18 adds native support for UUIDv7. A sortable UUID variant that plays nicely with indexes. Here's what that means for you.",
+  tags: ["postgres", "performance"],
+};
+
+PostgreSQL 18 is on the horizon, with beta testing now underway. Among the many improvements in this release is support 
+for UUIDv7. A UUID variant that is sortable and plays nicely with indexes. In this post, we'll discuss UUIDs in general, why UUIDv7 is so useful and how you'll want to use it in Postgres.
```

Comment:
all uuid are sortable? the main difference with v7 is that those are generated in a strictly monotonic order, which makes them much friendlier to some kind of indexes that don't play well with something like random insertion (like btree, but it likely wouldn't matter for some other kind of indexes)

Reply:
ok, thats fair. I meant "meaningfully sortable", since sorting random 128bits isn't all that helpful :)

Reply:
agreed :) but I think it's important to point out that the non meaningful sort order is only a problem for btree (although I don't think that many people use indexes with eg hash index method).